custom_components/opnsense/switch.py

```
@@ -10,241 +11,292 @@
 
-    cem = CoordinatorEntityManager(
-        hass,
-        hass.data[DOMAIN][config_entry.entry_id][COORDINATOR],
-        config_entry,
-        process_entities_callback,
-        async_add_entities,
+
+async def _compile_static_switches(
+    config_entry: ConfigEntry,
+    coordinator: OPNsenseDataUpdateCoordinator,
+    state: Mapping[str, Any],
```

Ruff

Unused function argument: `state`

(ARG001)